pkgs/applications/networking/browsers/ephemeral/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv
+, fetchFromGitHub
+, desktop-file-utils
+, gettext
+, glib
+, gtk3
+, hicolor-icon-theme
+, libgee
+, meson
+, ninja
+, pantheon
+, pkgconfig
+, python3
+, webkitgtk
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "ephemeral";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "cassidyjames";
+    repo = "ephemeral";
+    rev = version;
+    sha256 = "1bdqi1k1x2aywi2jm610qil2k5xn2kbs2nni55c5pgilq0sd7xnp";
+  };
+
+  nativeBuildInputs = [
+    gettext
+    meson
+    ninja
+    pantheon.vala
+    pkgconfig
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    desktop-file-utils
+    glib
+    gtk3
+    hicolor-icon-theme
+    libgee
+    pantheon.granite
+    python3
+    webkitgtk
+  ];
+
+  postPatch = ''
+    chmod +x meson/post_install.py
+    patchShebangs meson/post_install.py
+  '';
+
+  meta = with stdenv.lib; {
+    description = "The always-incognito web browser";
+    homepage = https://github.com/cassidyjames/ephemeral;
+    maintainers = with maintainers; [ kjuvi ] ++ pantheon.maintainers;
```

Comment:
Does it really makes sense to add all pantheon maintainers to ephermal as well?
Otherwise the PR looks good to me.

Reply:
Hmm I was told to do so for another PR for an app which came from [elementary app center](https://appcenter.elementary.io/). I think it was @worldofpeace suggest to do this.

Reply:
I have a lot of reasons to do this, and it's mostly useful for me to have `pantheon.maintainers` in there because I'm responsible for these packages.

Reply:
Alright. Thanks for the explanation.